### DISCUSSION TITLE
The documentation about Enzyme Induction‌?

### DISCUSSION THREAD

**Discussion by KGmax** [Open-Systems-Pharmacology/Forum#1590] [Questions & Problems] [answered]
Hello All,
I am learning the documentation about DDI. https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-defining-inhibition-induction-processes. But isn't there something missing from the file?
![酶诱导](https://github.com/Open-Systems-Pharmacology/Forum/assets/107936033/25aae5f6-15ee-49ad-bf94-846ae4e54d34)

Thanks.

**Comment by Yuri05** [comment, marked as answer]
True, it's an error in the documentation. Issue created (https://github.com/Open-Systems-Pharmacology/docs/issues/226)
